cleanlab/filter.py

```diff
@@ -427,18 +427,19 @@ class 0, 1, ..., K-1.
 
 
 def _find_label_issues_multilabel(
-    labels: list,
-    pred_probs: np.ndarray,
+    labels: LabelLike,
+    pred_probs: npt.NDArray["np.floating[T]"],
     return_indices_ranked_by: Optional[str] = None,
-    rank_by_kwargs={},
+    rank_by_kwargs:Dict={},
     filter_by: str = "prune_by_noise_rate",
     frac_noise: float = 1.0,
     num_to_remove_per_class: Optional[int] = None,
-    min_examples_per_class=1,
+    min_examples_per_class: int=1,
     confident_joint: Optional[np.ndarray] = None,
     n_jobs: Optional[int] = None,
     verbose: bool = False,
-) -> np.ndarray:
+) -> npt.NDArray[Union[np.bool_, np.int_]]:
+    #TODO: add docstring in the same format as other functions with args and returns
```

Comment:
Can I add to-dos wherever the doc doesn't have the same format as the other functions?

Reply:
Are you referring to the "shape"-format?

Reply:
The format of doc itself.

Reply:
Ohh right, now I see what you mean! Thank you for pointing this out!

That's not related to this PR, so you shouldn't add this particular to-do in this PR.
Feel free to [open an issue](https://github.com/cleanlab/cleanlab/issues/new/choose)!
If you find other instances, you can add them to the same issue.

We want to address these kinds of docstrings in a separate PR.